src/coreclr/tools/aot/ILCompiler.Compiler/IL/ILImporter.Scanner.cs

```diff
@@ -170,6 +170,9 @@ public DependencyList Import()
 
             }
 
+            _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.MemCpy), "Can be used by multiple IL opcodes implicitly");
+            _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.MemSet), "Can be used by multiple IL opcodes implicitly");
```

Comment:
@MichalStrehovsky What is the best way to do this?

Reply:
Probably to use RyuJIT as a scanner 🙂

Reply:
Ok, the next best way (also, using RyuJIT as a scanner would only help if optimizations cannot introduce calls to these helpers).

My concern is that it does not make sense to add an explicit dependency on these two helpers for every method, overhead and a lot of noise in the dependency graph.

Reply:
We use `RuntimeExport` for existing helpers that we do not model accurately. For example, look for `RhTypeCast_CheckCastAny`. I think we should use this pattern here as well.

Reply:
Ah I see, addressed